lib/internal/main/test_runner.js

```diff
@@ -13,7 +13,8 @@ const {
     ERR_INVALID_ARG_VALUE,
   },
 } = require('internal/errors');
-const { NumberParseInt } = primordials;
+const { NumberParseInt, RegExpPrototypeExec, StringPrototypeSplit } =
+  primordials;
```

Comment:
```suggestion
const {
  NumberParseInt,
  RegExpPrototypeExec,
  StringPrototypeSplit,
} = primordials;
```
this way future changes don't cause as much diff churn

Reply:
thanks, done